beacon_node/network/src/beacon_processor/worker/mod.rs

```diff
@@ -25,7 +25,7 @@ impl<T: BeaconChainTypes> Worker<T> {
     /// Send a message to `sync_tx`.
     ///
     /// Creates a log if there is an internal error.
-    fn send_sync_message(&self, message: SyncMessage<T::EthSpec>) {
+    fn send_sync_committee_message(&self, message: SyncMessage<T::EthSpec>) {
```

Comment:
This was a spurious find-and-replace on Sean's branch that was fixed in this commit https://github.com/sigp/lighthouse/pull/2321/commits/22da62325ce558e22218c9653c38c6c305ce235a

Reply:
Cherry picked the commit. Thanks for pointing it out!